### ISSUE TITLE
No audio in qutebrowser, amixer error

### ISSUE THREAD

**Issue by jb55** [NixOS/nixpkgs#157546]
### Describe the bug

Not sure when the started occuring, but my volume control (amixer) and audio in qutebrowser stopped working. Applications like spotify and mpv still have audio. I switched to pamixer and that fixed my volume controller, but I still have no audio in qutebrowser.

*insert audio not working on linux meme here*

### Steps To Reproduce
1. run `amixer`

output: 

```
ALSA lib control.c:1464:(snd_ctl_open_conf) Unknown field libs
amixer: Mixer attach default error: Invalid argument
```

I'm on nixpkgs/nixos-unstable @ 5bb20f9dc70e9ee16e21cc404b6508654931ce41

**Comment by liketechnik** [comment]
You can find the cause/potential fix and more details in #157352

**Comment by jb55** [comment]
@liketechnik thanks! I was looking for this.

**Comment by rnhmjoj** [comment]
This should have been fixed by #157631, currently in staging.
For tracking the progress see: https://nixpk.gs/pr-tracker.html?pr=157631.
